docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
+:::
 
-Enterprise plans are custom to each customer, but typically include:
+::: details Developing a workflow with test events in the Pipedream workflow builder
 
-- Custom invocations, users, and orgs
-- SSO support (GSuite and [Okta](/orgs/sso/okta/))
-- [A 99.95% uptime SLA](https://pipedream.com/sla)
-- Support for multiple orgs. For example, you might want to separate workflows owned by different teams.
-- Premium support: onboarding, ongoing training, and anything your team needs to be successful
-- Custom, prioritized [component](https://pipedream.com/docs/components/) development. If you need new sources or actions, The Pipedream team will develop them for you and prioritize that work on our [component backlog](https://github.com/PipedreamHQ/pipedream/projects/1).
+0 credits are incurred. Developing and testing your workflows is free. 
 
-To discuss an Enterprise plan, please contact the Pipedream Sales Team [using the **Contact Sales** form on our Pricing page](https://pipedream.com/pricing).
+Execution time used to develop a workflow in the builder does not count towards your credit usage.
 
-## Definition of Terms
+:::
 
-Pipedream uses a number of terms to describe platform metrics and details of our plans. See the definitions of key terms below.
+::: details An active standard workflow that isn't triggered at all in a billing period
 
-### Invocations
+0 credits are incurred. Pipedream only charges credits for workflow executions.
 
-Pipedream counts an **invocation** each time a workflow or event source is triggered by an incoming event.
+:::
 
-Pipedream increments the count of invocations by one for each incoming event, regardless of the number of steps in your workflow. For example, if you send an HTTP request to a workflow with five steps, Pipedream will count that as one invocation.
+::: details A workflow with 512 megabytes of memory running for 5 seconds
 
-Moreover, if you have a workflow triggered by a cron job running once a minute for the entire day, that will incur 1,440 invocations (60 minute \* 24 hours). For cron-triggered workflows or event sources, remember that you can always reduce the frequency to reduce your invocations.
+2 credits are incurred, because the workflow executed for a total of 5 seconds at 512 megabytes of memory.
+:::
 
-If an event emitted by an event source triggers a single workflow, that will count as **two** invocations: one for the source, and one for the workflow. In other words, source and workflow execution is distinct: each counts invocations on its own.
 
-Your workflow's [memory settings](/workflows/settings/#memory) also impact the number of invocations you're charged for each workflow execution. [Read more here](#how-does-workflow-memory-affect-billable-invocations).
+#### Source Credit Usage
 
-#### Scenarios
+The first credit used during a [source's](/docs/sources/) execution is included for free.
```

Comment:
I was reading this section and thinking about how we could also educate people on sources again (since most people will have no idea what it means still). wdyt about this?

> Many workflow triggers run as separate Pipedream resources called [sources](/docs/sources). You can view all sources at [https://pipedream.com/sources](https://pipedream.com/sources). Since they're separate resources, they also incur credits. When you build a workflow and use one of our public triggers, **the first credit (30 seconds of runtime) for every execution is free**. Most sources run under 30 seconds, so you normally won't see credits charged to sources.

> You'll only be charged source credits in the following conditions:

> * Sources that run longer than 30 seconds will be charged after the first, free credit. For example, if a source runs for 45 seconds, it will be charged 1 credit. If it runs for 65 seconds, it will be charged two credits.
> * If you [build your own private sources](https://pipedream.com/docs/components/quickstart/nodejs/sources/), you'll be charged for all credits.
> * If a source doesn't trigger a workflow, it's called [an independent source](https://pipedream.com/docs/workflows/steps/triggers/#dependent-and-independent-sources). Independent sources will be charged for all credits.